### DISCUSSION TITLE
Is there an example of changing tab colours based on a shell script?

### DISCUSSION THREAD

**Discussion by aclarknexient** [kovidgoyal/kitty#6983] [General]
I'm pretty new to Kitty, and I really like its extremely customizable features. I was hoping to mimic a feature I use at work to separate dev/qa/prod environments in AWS. I documented it in a gist that probably explains what I want to do much better: https://gist.github.com/aclarknexient/84ebe33c1879f92168530487528802c4

<img width="1078" alt="Screenshot 2024-01-08 at 12 31 58" src="https://github.com/kovidgoyal/kitty/assets/69362332/7a25a07a-6034-4609-a412-8769058d96d6">

Is there an example of something similar in Kitty that I can use as a starting point? I found this discussion but was a bit lost: https://github.com/kovidgoyal/kitty/discussions/4447

Thank you for a fantastic piece of software!

**Comment by tg0h** [comment]
if you just want to change your tab colour, you can use this command `kitty @set-colors active_tab_background=red`

<img width="599" alt="image" src="https://github.com/kovidgoyal/kitty/assets/7954426/84991b05-1e1e-4447-bdc9-08f205d7faff">


the discussion you linked is only needed if you want to customize your tab bar more
